articles/getting_started/content_pipeline/content_builder_project.md

```diff
@@ -177,6 +177,72 @@ You should not need to touch anything else.
 >
 > See the [MS Documentation on Built targets](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-targets) for reference.
 
+## Platform specific additions - Android / iOS
+
+For platforms where the content is required to be packaged inside the output bundle, there are some additional steps required to ensure the built content is including during the Build/Publish step.
+
+Each platform has specific requirements and require the built content to be available prior to processing the project itself.
+
+> [!NOTE]
+> Also check out the dedicated [Packaging Games advice](../packaging_games.md) published by the MonoGame team.
+
+### Android
+
+For Android, a specific `Include` section is needed in the project's `csproj` project definition, as follows:
+
+> [!NOTE]
+> Android uses a special `AndroidAsset` tag to identify content to include.
+
+```xml
+<ItemGroup>
+    <AndroidAsset Include="$(OutputPath)Content\**\*">
+        <Link>Content\%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </AndroidAsset>
+</ItemGroup>
+```
+
+> [!NOTE]
+> If you change the location where the Content Builder project outputs content to, the paths above will need to be updated to the new location to find the content to include.
+
+At build time, the "BuildContent" task will run and execute the Content Builder, then with the output from the builder, the Android solution will automatically identify and package the content within the `APK` as per Google's packaging requirements.
```

Comment:
`APK/AAB`

note: you can't upload `apk` files to google any more, but you can for other third party stores.